ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -142,9 +142,44 @@ objects:
           oc create -f /tmp/cluster/insights-live.yaml || true
         fi
 
-        # set up cloud-provider-specific env vars
-        export KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
+        # set up SSH for the e2e tests + for this script
+        function setup_ssh_bastion() {
+          echo "Setting up ssh bastion"
+          mkdir -p ~/.ssh || true
+          cp "${KUBE_SSH_KEY_PATH}" ~/.ssh/id_rsa
+          chmod 0600 ~/.ssh/id_rsa
+          if ! whoami &> /dev/null; then
+            if [ -w /etc/passwd ]; then
```

Comment:
prefer `[[` over `[`

Reply:
I just moved the code from below, I did not want to change it...